docs/guide/api-environment-frameworks.md

```diff
@@ -106,7 +106,7 @@ const server = await createServer({
 
 // You might need to cast this to RunnableDevEnvironment in TypeScript or
 // use isRunnableDevEnvironment to guard the access to the runner
-const environment = server.environments.node
+const server = viteServer.environments.server
```

Comment:
```suggestion
const serverEnvironment = viteServer.environments.server
```
I think `serverEnvironment` would be a name if we want to be explicit about which environment it refers to. `server` feels a bit confusing with `viteServer` to me.

Reply:
Exactly my thought.